landa/api.py

```diff
@@ -112,3 +113,13 @@ def legal():
 		"privacy_policy": rules.privacy_policy,
 		"imprint": rules.imprint,
 	}
+
+
+@frappe.whitelist(allow_guest=True, methods=["GET"])
+def change_log(from_datetime: str = None):
+	"""Return a list of version logs of documents created/updated/deleted after the given datetime."""
+	if not from_datetime:
+		return []
```

Comment:
```suggestion
def change_log(from_datetime: str):
	"""Return a list of version logs of documents created/updated/deleted after the given datetime."""
	if not isinstance(from_datetime, str):
		raise TypeError("from_datetime must be a string")
```

- Feels unintuitive to allow `from_datetime` to be missing if it is required to receive a meaningful response. Better raise an error if the request is wrong.
- Validate that the parameter is of correct type.

Reply:
Also, do we need to `frappe.db.escape(from_datetime)` or does pypika take care of this?

Reply:
@barredterra pypika escapes any string passed to it